CMakeLists.txt

```diff
@@ -63,6 +63,12 @@ if(BUILD_POSTEXEC)
   endif()
 endif()
 
+### Switch RUNTIME DESTINATION DIR between bin and exec
+set(exec_dir bin)
```

Comment:
@chan-hoo Is the default setting of exec_dir "OFF" in cmake configuration?

Reply:
`exec_dir` takes a default value of `bin` unless it's overridden on the command line. 

Maybe it should be made a top-level option with `set(EXEC_DIR bin CACHE STRING "Directory where executables are placed")`

Reply:
Once again, I didn't fully read. EMC_EXEC_DIR has no default value. So, it's off by default.

Reply:
@WenMeng-NOAA, as @kgerheiser said, the default values of `exec_dir` and `EMC_EXEC_DIR` are `bin` and `OFF`, respectively. Once `EMC_EXEC_DIR` is on, the value of `exec_dir` is changed to `exec`. I've tested the both options and I've confirmed that they worked correctly.

Reply:
@kgerheiser Yes, I am meaning EXC_EXEC_DIR. Thanks for clarification.